src/aics-image-viewer/components/App/ChannelUpdater.tsx

```diff
@@ -60,6 +59,20 @@ const ChannelUpdater: React.FC<ChannelUpdaterProps> = ({ index, channelState, vi
     [color]
   );
 
+  const { controlPoints, ramp, useControlPoints } = channelState;
+  useImageEffect(
+    (currentImage) => {
+      if (useControlPoints && controlPoints.length < 2) {
```

Comment:
what is this if condition checking for? is it for before any channel data has arrived, or something like that? I see it was there in the previous implementation too

Reply:
Looks like this triggers only when the user drags a control point all the way to the left, then all the way to the right, thereby deleting all but one point. The effect is that the transfer function is left in its last valid state rather than getting zeroed out when there's only one control point. Does this behavior make sense?

Reply:
I guess?  It kinda sounds like something that could be handled inside the tfeditor but at least I get the scenario where it could happen.